src/request.m

```diff
@@ -295,8 +295,11 @@ - (void) _updateWindow: (NSInteger) delta withBlock: (void (^)()) block {
       window_out_queue_ = [NSMutableArray arrayWithCapacity: 16];
 
     // Retry on positive window_out
+    __weak __typeof(self) weakSelf = self;
```

Comment:
Does this mean that a final `_updateWindow` message will be passed to the `ISpdyRequest` object? The one that nils the `window_out_queue_` variable, thus breaking the circular reference? If yes, this means the request has at least one reference held by the caller of `_updateWindow`. If not, it means that the last `_updateWindow` that enqueues the block in discussion will create a leak.